examples/allocation-endpoint/client/main.go

```diff
@@ -0,0 +1,152 @@
+// Package main runs an Allocation Endpoint client with configurations for benchmarking
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	_ "embed"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"sync"
+	"time"
+
+	pb "agones.dev/agones/pkg/allocation/go"
+	"golang.org/x/oauth2"
+	"google.golang.org/api/idtoken"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	port = "443"
+)
+
+var (
+	clientCount    = flag.Int("clientCount", 10, "Number of clients to perform allocations")
```

Comment:
In the docs clientCount is set to 1 and perClientCalls is set to 1 and runs is set to 1. Would it make sense for the defaults here to use those values so that the docs can be simplified to not need to pass those parameters?